perfmetrics/scripts/ls_metrics/directory.proto

```diff
@@ -0,0 +1,12 @@
+syntax="proto3";
+
+package directory_proto;
+
+message Directory{
```

Comment:
nit: spacing
space between Directory and {
and for every property space between = like "name = 1;